frontend/src/components/about/AboutUs.js

```diff
@@ -11,11 +11,13 @@ class AboutUs extends React.Component {
                         <div className="col-xs-12 col-md-4">
                             <div className="card mt-5 mr-5 ml-5">
                                 <div class="card-body text-center">
-                                    <img className="rounded-circle" src="./aboutUs-Jessica.jpg" alt="..." />
+                                    <a href="https://www.linkedin.com/in/jessica-shortz-02572831/" target="_blank" style={{ textDecoration: 'none' }}>
```

Comment:
I like the idea of having LinkedIn link here but maybe it can create inconsistencies since is only in one element. thoughts on that?

Reply:
My thinking was that we could click on one link for demo if wanted. Ideal world, would add similar link to all pictures.